ibctest/ibc_test.go

```diff
@@ -57,6 +58,23 @@ func TestRelayerDocker(t *testing.T) {
 	ibctestConformance(t, rf)
 }
 
+// TestRelayerDocker runs the ibctest conformance tests against
+// the current state of this relayer implementation built in docker.
+// Relayer runs using the event processor.
+func TestRelayerDockerLegacyProcessor(t *testing.T) {
+	relayeribctest.BuildRelayerImage(t)
```

Comment:
It seems like both of these should have `t.Parallel()` for when someone runs a broad `go test .` in the `ibctest` module. That would have no change against using the make targets from CI.

Reply:
done